src/website/src/pages/ec/components/page-banner/docs/usage.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  A page banner displays a prominent message and related optional actions.
+  Banners are displayed on the homepage below the site header of EC sites. It
+  conveys an important message to your audience. Page banners are a good way to
+  channel your visitors to complete a specific task (ex: register to an event,
+  apply for funding... ). The user can choose to ignore them or interact with
+  them at any time.
+</Paragraph>
+
+## Anatomy
+
+Banners should have a focused message and provide users with clear actions to take. They should be placed at the top of a layout or screen. When scrolling, banners typically move with content and scroll off the screen. Only one banner should be shown at a time.
```

Comment:
```suggestion
Banners should be placed at the top of a webpage. It can have a combination of a photo or graphics with copy and call to action. Depending on the need, not all elements are mandatory.
```

Reply:
"copy" is ambiguous for non writer and non native english.
I would suggest to use the word "text" instead.